spec/system/post_selection_fast_edit_spec.rb

```diff
@@ -3,46 +3,54 @@
 describe "Post selection | Fast edit", type: :system do
   let(:topic_page) { PageObjects::Pages::Topic.new }
   let(:fast_editor) { PageObjects::Components::FastEditor.new }
+
```

Comment:
This is unrelated to the fix, but related to the feature nonetheless. I did that refactoring while exploring / understanding the issue.